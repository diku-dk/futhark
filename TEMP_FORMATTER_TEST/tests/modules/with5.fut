-- Refinement of a parametric may expand it.
-- ==

module type has_t = { type t 'a }
module type has_t' = has_t with t '^a = a
