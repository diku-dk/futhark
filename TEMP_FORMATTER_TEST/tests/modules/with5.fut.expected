db80eb11f4679b3ba13c3590e199b8b6
